tox.ini

```diff
@@ -111,6 +111,7 @@ setenv =
     PYTHONPATH = {envtmpdir}/mongo-python-driver
 commands =
     git clone --depth 1 --branch master https://github.com/mongodb/mongo-python-driver.git {envtmpdir}/mongo-python-driver
+    pip install -e {envtmpdir}/mongo-python-driver
```

Comment:
I remember this caused some problems when we were on a call. Let's remove it?

Reply:
Yeah, my impression was that this line should be there as well.